Source/WTF/wtf/PlatformHave.h

```diff
@@ -1567,3 +1567,7 @@
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 130000)
 #define HAVE_LS_SERVER_CONNECTION_STATUS_RELEASE_NOTIFICATIONS_MASK 1
 #endif
+
+#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000
```

Comment:
Shouldn't this be MAX_ALLOWED? So that we don't unnecessarily complicate building for 10.13 using 10.14 SDK.

Reply:
Great catch. It's declared in the 14 SDK, so we should be checking SDK version and not deployment target.